src/app/pages/profile/profile-sidebar/profile-sidebar.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="card profile-sidebar-upper">
+  <div class="profile-image-container">
+    <ngx-spinner></ngx-spinner>
+
+    <img
```

Comment:
@MandelaK Good work man, However it can be really efficient if users can not upload media files that are not images. We can achieve this by adding an `accept` attribute to the img tag.

Reply:
Thanks for pointing this out, I have fixed it